### ISSUE TITLE
trial_image_slider_response() not saving image file name 

### ISSUE THREAD

**Issue by perfors** [djnavarro/jaysire#3]
As far as I can tell, this function does not save the name of the image stimulus in the data file. It does save the slider response though. I am not sure if this generalises to other image functions or other slider functions, I ran out of time to check.

**Comment by kupiqu** [comment]
It seems an upstream issue, doesn't it?

https://github.com/jspsych/jsPsych/issues/689

**Comment by jodeleeuw** [comment]
Yeah I think this should be fixed in the jspsych plugin. It's an easy fix but the release cycle is a bit slow right now...
